Tests.Integration.Presentation.Web/Users/UserTest.cs

```diff
@@ -119,6 +121,47 @@ public async Task Cannot_Get_Users_Where_User_Has_StakeHolder_Or_ApiAccess_If_No
             Assert.Equal(HttpStatusCode.Forbidden, result.StatusCode);
         }
 
+        [Fact]
+        public async Task Delete_User()
+        {
+            var userRole = OrganizationRole.LocalAdmin;
+
+            var (_, userId, organization) = await CreatePrerequisitesAsync(userRole);
+            var name = A<string>();
+
+            await RightsHelper.AddUserRole(userId, organization.Id, RightsType.OrganizationUnitRights, name);
+            await RightsHelper.AddUserRole(userId, organization.Id, RightsType.ItContractRights, name);
```

Comment:
what about DPR rights an all of the other things which are removed in organizationService.deleteUser?